mmpose/core/evaluation/top_down_eval.py

```diff
@@ -23,7 +23,7 @@ def _calc_distances(preds, targets, normalize):
     N, K, _ = preds.shape
     distances = np.full((K, N), -1, dtype=np.float32)
     eps = np.finfo(np.float32).eps
-    mask = (targets[..., 0] > eps) | (targets[..., 1] > eps)
+    mask = (targets[..., 0] > -eps) & (targets[..., 1] > -eps)
```

Comment:
is this a bug fix?

Reply:
Yes. Keypoints whose coordinates are  (x, 0) or (0, y) should be valid.